Include missing headers for PGPLOT, so that GCC 14+ do not complain
about them anymore.

Author: Jan Reuter (jan@zyten.de)

diff --color -Naur pgplot.orig/sys/grfileio.c pgplot/sys/grfileio.c
--- pgplot.orig/sys/grfileio.c	1995-06-19 22:41:42.000000000 +0200
+++ pgplot/sys/grfileio.c	2025-12-13 23:24:46.613167824 +0100
@@ -20,6 +20,8 @@
 #include <stdio.h>
 #include <sys/types.h>
 #include <fcntl.h>
+#include <string.h>
+#include <unistd.h>
 
 #ifdef PG_PPU
 #define GROFIL grofil_
diff --color -Naur pgplot.orig/sys/grtermio.c pgplot/sys/grtermio.c
--- pgplot.orig/sys/grtermio.c	1994-12-29 21:00:33.000000000 +0100
+++ pgplot/sys/grtermio.c	2025-12-13 23:24:27.690856797 +0100
@@ -7,6 +7,9 @@
 
 #include <stdio.h>
 #include <termios.h>
+#include <string.h>
+#include <unistd.h>
+#include <fcntl.h>
 
 #ifdef PG_PPU
 #define GROTER groter_
